public/app/features/explore/Logs.tsx

```diff
@@ -346,6 +367,27 @@ export class UnthemedLogs extends PureComponent<Props, State> {
           />
         )}
 
+        {hasUnescapedNewlines && (
+          <MetaInfoText
+            metaItems={[
+              {
+                label: 'Your logs might have incorrectly escaped newlines',
```

Comment:
maybe, as it's also about tabs:

```suggestion
                label: 'Your logs might have incorrectly escaped content',
```